x/txfees/types/keys.go

```diff
@@ -10,11 +14,28 @@ const (
 	// RouterKey is the message route for slashing.
 	RouterKey = ModuleName
 
-	// QuerierRoute defines the module's query routing key.
+	// FeeCollectorName the module account name for the fee collector account address.
+	FeeCollectorName = "fee_collector"
+
+	// NonNativeFeeCollectorName the module account name for the alt fee collector account address (used for auto-swapping non-OSMO tx fees).
+	NonNativeFeeCollectorName = "non_native_fee_collector"
+
+	// QuerierRoute defines the module's query routing key
 	QuerierRoute = ModuleName
 )
 
 var (
 	BaseDenomKey         = []byte("base_denom")
 	FeeTokensStorePrefix = []byte("fee_tokens")
+
+	// AddressStoreKeyPrefix prefix for account-by-address store
+	AddressStoreKeyPrefix = []byte{0x01}
+
+	// param key for global account number
+	GlobalAccountNumberKey = []byte("globalAccountNumber")
 )
+
+// AddressStoreKey turn an address to key used to get it from the account store
+func AddressStoreKey(addr sdk.AccAddress) []byte {
+	return append(AddressStoreKeyPrefix, addr.Bytes()...)
+}
```

Comment:
None of this is actually used from what I can tell. Why was this added?

Reply:
Hmm I believe this is vestigial from when I was first migrating the relevant ante handler into our txfees module - removing in next commit